R/render.R

```diff
@@ -4,7 +4,10 @@
 #' (knitted from R Markdown) and JavaScript code to render slides.
 #' \code{tsukuyomi()} is an alias of \code{moon_reader()}.
 #'
-#' Tsukuyomi is a genjutsu to trap the target in an illusion on eye contact.
+#' @details Tsukuyomi is a genjutsu to trap the target in an illusion on eye contact.
+#' @details If you are unfamiliar with CSS, please see the
```

Comment:
Sure. The Github review model is very useful but I can understand that it may look strange to you at the moment. Basically the reviewer can ask for required changes (such as in this case), and the PR cannot be merged until the lines under which the reviews were written have been updated (not strictly true, but that is the idea).